compose/compose-file/index.md

```diff
@@ -882,6 +882,73 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
+
+#### Short syntax
+
+The short syntax variant only specifies the secret name. This grants the
+container access to the secret and mounts it at `/run/secrets/<secret_name>`
+within the container. The source name and destination mountpoint are both set
+to the secret name.
+
+> **Warning**: Due to a bug, using the short syntax currently mounts the secret
```

Comment:
I think we have to include it, otherwise people on 1.13.1 will have a hell of a hard time figuring out what is going on (like I did, and @nathanleclaire did). I'm OK with putting the version on it. We can't assume everyone will instantly update to 1.13.2, or that people who aren't on 1.13 yet will go straight to 1.13.2. In fact, I don't think we **can** remove the note as long as 1.13.1 is a supported configuration. @thaJeztah ?